services/web/server/src/simcore_service_webserver/activity/handlers.py

```diff
@@ -0,0 +1,49 @@
+from ..login.decorators import login_required
+import aiohttp
+from urllib.parse import quote
+import asyncio
+import json
+
+@login_required
+async def get_status(request: aiohttp.web.Request):
+    async with aiohttp.ClientSession() as session:
```

Comment:
For the moment this is ok but you should NOT create your own session in the handler but instead get it from the application:
```python
from servicelib.client_session import get_client_session

@login_required
￼async def get_status(request: aiohttp.web.Request):
    session = get_client_session(request.app)
    ...

```
Nonetheless, this function will be available only when PR #1110 is merged

Reply:
noted

Reply:
You can already use this feature ... the changes are in. Let me know if you need some assistance